### ISSUE TITLE
Unable to transfer airdrops

### ISSUE THREAD

**Issue by JohnAlf** [greymass/eos-voter#607]
The transfer of PATR tokens from one account to another seemed to be working well. The transaction was made and transferred to the airdrop and signed without issue. It imported back in the Watch, but when I tried to broadcast it got the attached error. Tried with PATR, KARMA, IQ, BKT and WECASH
![PATR-transferError](https://user-images.githubusercontent.com/40725765/56515701-82ca0e80-6506-11e9-8aa8-661ac83b161c.jpg)

**Comment by JohnAlf** [comment]
Found the problem. I had pasted the quantities to transfer from an Excel sheet that was formatted with commas. (1,243.00). On the next page it took that and made a zero quantity, which I didn't notice. Ends up that I was trying to confirm a transfer transaction that had no quantity, so it bombed. The error message didn't explain, "Hey dufus, you just tried to transfer a 0 quantity," do I didn't get it at first.
